regression-tests.recursor-dnssec/test_LockedCache.py

```diff
@@ -102,4 +102,4 @@ def testMX(self):
         self.assertRRsetInAnswer(res, expected2)
         self.assertMatchingRRSIGInAnswer(res, expected2)
         ttl2 = self.getCacheTTL()
-        self.assertAlmostEqual(ttl1, ttl2, 1)
+        self.assertAlmostEqual(int(ttl1), int(ttl2), 1)
```

Comment:
Dare I suggest making sure that `getCacheTTL()` returns an integer value instead.

Reply:
Better indeed